Eatery Blue/UI/EateryScreen/EateryViewController.swift

```diff
@@ -23,6 +23,7 @@ class EateryViewController: UIViewController {
     let navigationView = EateryNavigationView()
     let scrollView = UIScrollView()
     let stackView = UIStackView()
+    let spinner = UIActivityIndicatorView(style: .large)
     var categoryViews: [MenuCategoryView] = []
```

Comment:
Usually these should be sorted alphabetically, but if it isn't in the rest of the codebase then it's fine.